main.go

```diff
@@ -85,6 +86,14 @@ func main() {
 	// temporary for us, for release want info level - todo make configurable via cli?
 	zerolog.SetGlobalLevel(zerolog.DebugLevel)
 
+	//  start mqtt certificate verification routine
+	cs := oauth.NewCertificateService(logger, env, "/opt/autopi/client.pem")
```

Comment:
I would just provide the cert directory here.

Reply:
code updated